book/src/queries/README.md

```diff
@@ -0,0 +1,10 @@
+# Queries
+
+PRQL queries are composed of a sequence of statements (a _pipeline_) that modify
+or _transform_ data as it is passed from statement to statement.
```

Comment:
Technically they're not statements... They're transforms / expressions because they evaluate to a value. could we maybe replace `statement` with `transform`? Or is that a lot of `transform`?

Reply:
I'll fix.